src/braket/tasks/gate_model_quantum_task_result.py

```diff
@@ -151,6 +151,36 @@ def get_compiled_circuit(self) -> Optional[str]:
         else:
             return None
 
+    def partial_measurements(self, qubits: Iterable[int]) -> np.ndarray:
```

Comment:
General comment: I don't think "partial measurement" is a term we would want to expose in the API. Just "measure", along with the list of qubits to measure.

Reply:
Got it. So do you think it would be better to add an optional filter kwarg (`qubits` maybe?) to the existing functions, instead of introducing new ones?

Reply:
That would make more sense to me, yes. But I agree with Kshitij that a measure instruction in the circuit itself is likely to be a more useful way to expose this. The post-processing step doesn't do much that the user can't already do today fairly easily (in fact, these new methods have one-line implementations).